content/en/synthetics/api_tests/http_tests.md

```diff
@@ -213,8 +213,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level. 
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the timeout defined in the test (default is set to 60s).
```

Comment:
```suggestion
  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
```
Should we link to the API docs for increasing the timeout value?

Reply:
timeout for API tests is by default set to 60s and it is also the max value. Users can only reduce it.
looks like timeout are not in the docs. I'll add it too